security/s2a/internal/crypter/aeadcrypter.go

```diff
@@ -0,0 +1,18 @@
+package crypter
+
+// S2AAeadCrypter is the interface for an AEAD cipher used by the S2A record
+// protocol.
+type S2AAeadCrypter interface {
+	// Encrypt encrypts plaintext, computes the tag of plaintext, and appends
+	// the resulting ciphertext to dst, returning the updated slice. dst and
```

Comment:
Sorry for the back and forth on this. Let's keep the documentation the same as ALTS, after your latest fix. Basically, we might not want to commit that the result is append to `dst` since we might do something different in some implementations of the interface.